dcat/index.html

```diff
@@ -1889,7 +1885,7 @@ <h4>Property: access service</h4>
             <table class="definition">
                 <thead><tr><th>RDF Property:</th><th><a href="http://www.w3.org/ns/dcat#accessService">dcat:accessService</a></th></tr></thead>
                 <tbody>
-                <tr><td class="prop">Definition:</td><td>A site or end-point that gives access to the distribution of the dataset</td></tr>
+                <tr><td class="prop">Definition:</td><td>A data service that gives access to the distribution of the dataset</td></tr>
```

Comment:
how about just 'service' or 'end-point'?

Reply:
The access service is a data service, which is not a site but an API. Defining it as a "service" isn't offering much in the way of information. Why not be more helpful and say that it is a data service?